components/layermanager/layermanager.service.js

```diff
@@ -713,9 +711,12 @@ export default function (
    * @function isLayerInResolutionInterval
    * @memberOf HsLayermanagerService
    * @param {Ol.layer} lyr Selected layer
-   * @description Test if layer (WMS) resolution is within map resolution interval
+   * @description Test if layer (WMS) resolution is NOT within map resolution interval. True = out of the interval
```

Comment:
I think it should have been "Map resolution is within layers resolution interval minResolution <= current <= maxResolution"

Reply:
the condition
```
lyr.getMinResolution() >= cur_res && cur_res <= lyr.getMaxResolution()
```
Seems to be the correct one.

Reply:
This ones bit confusing. The logic while using the return vaule is twisted:
`grayed: me.isLayerInResolutionInterval(layer)` .. should not be grayed if in resolution

Reply:
Made some changes, seems to work fine now.
2a93397e3017698934abee272a92e293aee594a9